modules/cnf-working-with-fast-event-notifications-api.adoc

```diff
@@ -0,0 +1,283 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-working-with-fast-event-notifications-api_{context}"]
+= Working with the PTP fast event notifications REST API
+
+You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
+
+You can subscribe DU applications to PTP event notifications using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
+
+Status request are sent by making a REST API `PUT` call to `/subscriptions/status/<subscription_id>`. The API call is returned to the return URL specified in the subscription for the specified `<subscription_id>`.
```

Comment:
Sorry, I meant , the API call will return events via amqp bus to the subscribed address.

Reply:
NP - updated with "The API call returns events through the AMQP event bus to the subscribed address."